tcf_website/models/models.py

```diff
@@ -61,33 +73,62 @@ class Meta:
 
 
 class Subdepartment(models.Model):
-    """Subdepartment model.
+    """Subdepartment model. Represents a specific subject within a Department.
+    For example, "CREO" in the French department, "ENCW" in the English department,
+    or "CS - Computer Science" in the Computer Science department.
```

Comment:
Technically, "CS - Computer Science" is just the `str` representation, and the department is just "Computer Science". The `str` for CREO is "CREO - CREO" which indeed is wack, but there's a contradiction in the notes.

super pedantic lol but while I'm reviewing might as well mention it